src/operator/nn/dnnl/dnnl_sum.cc

```diff
@@ -23,10 +23,11 @@
  * \author Da Zheng
  */
 #include <iostream>
+#include <vector>
 
 #include "operator/operator_common.h"
 #include "dnnl_base-inl.h"
-#include "dnnl_ops-inl.h"
+#include "dnnl_sum-inl.h"
```

Comment:
Please uniform sum operator - class declaration (and its functions) should be in inl.h file. Please see how it is done in e.g. dnnl_eltwise* files.

Reply:
Is it right now or some additional changes are needed?

Reply:
I would do it within this PR.